website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 1
+description: Create one or more indexes on a collection to improve query performance and enable certain query features.
+---
+
+# `createIndexes`
+
+import CodeBlock from '@theme/CodeBlock';
+import CreateIndex from '!!raw-loader!../../guides/create-indexes/1-create-simple-index.request.js';
+import CreateUniqueIndex from '!!raw-loader!../../guides/create-indexes/2-create-unique-index.request.js';
+import CreateCompoundIndex from '!!raw-loader!../../guides/create-indexes/3-create-compound-index.request.js';
+import CreatePartialIndex from '!!raw-loader!../../guides/create-indexes/4-create-partial-index.request.js';
+
+<!-- Category for Command "Administrative, Diagnostic, and Utility Commands" should be organized per folder as present in the compatibility document -->
+
+<!-- Short description of the command and its usage. -->
+
+Create one or more indexes on a given collection to improve query performance or enable certain query features (e.g., full-text, vector, and TTL indexes).
+
+<!-- Syntax structure of the command -->
+
+<!-- markdownlint-disable MD033 -->
+
+This command accepts a collection name and an array of index specification documents.
+The index specification document specifies the fields to be indexed, the index name, and other options such as uniqueness, partial filter expressions, and expiration times for TTL indexes.
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>',
+  indexes: [
+    {
+      key: {
+        '<field1>': 1, // ascending
+        '<field2>': -1, // descending
+        '<field3>': 'text' // for full-text search
```

Comment:
Can you concatenate full-text search index with standard ascending and descending indexes? I think that's weird :thinking: this specification seems to indicate it's possible.